src/main/java/org/stellar/sdk/TimeBounds.java

```diff
@@ -61,12 +81,16 @@ public static TimeBounds fromXdr(org.stellar.sdk.xdr.TimeBounds timeBounds) {
 
 	@Override
 	public boolean equals(Object o) {
-		if (this == o) return true;
-		if (o == null || getClass() != o.getClass()) return false;
-
+		if (this == o) {
+			return true;
+		}
+		if (o == null || getClass() != o.getClass()) {
+			return false;
+		}
 		TimeBounds that = (TimeBounds) o;
-
-		if (mMinTime != that.mMinTime) return false;
```

Comment:
I think it's easier to read
```java
return mMinTime == that.mMinTime && mMaxTime == that.mMaxTime;
```

instead of:
```java
		if (mMinTime != that.mMinTime) {
			return false;
		}
		return mMaxTime == that.mMaxTime;
```

Reply:
Thanks, done.

Reply:
sorry, wait for me to fix the commit log